bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/dialogs/OrderImportDialog.java

```diff
@@ -513,6 +532,23 @@ private void applyScanner(String gtin, int diff, IArticle article) {
 		viewer.refresh();
 	}
 
+	private void executeMediorderPrintLabel(OrderElement orderElement, String gtin) {
+		ICommandService commandService = PlatformUI.getWorkbench().getService(ICommandService.class);
```

Comment:
Oben in dieser Classe wird ein `@Inject` gemacht, dass ist bereits e4 Style, du machst das e3 style, wovon wir eigentlich wegwollen. Bitte hol dir den ICommandService und auch den IHandlerService über `@Inject`